packages/protocol/src/interaction/FabricAccessControlManager.ts

```diff
@@ -71,34 +75,68 @@ export class AccessControlManager {
         clusterId: ClusterId,
     ) => boolean = () => true;
 
-    constructor(
-        aclList: AccessControl.AccessControlEntry[] = [],
-        extensionEntryAccessCheck?: (
+    constructor(fabric?: Fabric) {
+        if (fabric === undefined) {
+            this.#fabricIndex = FabricIndex.NO_FABRIC;
+            // Add the implicit default PASE ACL entry for the fabric
+            this.#aclList.push(ImplicitDefaultPaseAclEntry);
+        } else {
+            this.#fabricIndex = fabric.fabricIndex;
+        }
+    }
+
+    set fabricIndex(fabricIndex: FabricIndex) {
```

Comment:
Should this be removed?  The constructor sets `#fabricIndex` and this setter always throws InternalError